frontend/src/components/shared/execution-table.module.css

```
@@ -24,29 +26,34 @@
+  transition: color var(--t-fast) var(--ease);
+}
+
 .statusLabel {
   font-family: var(--font-mono);
   font-size: var(--fs-micro);
   white-space: nowrap;
 }
 
-.errorType {
+.okLabel {
+  composes: statusLabel;
```

Stylelint (17.14.0)

[error] Expected "statusLabel" to be "statuslabel" (value-keyword-case)

(value-keyword-case)